docker/test-server/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9.0-alpine
```

Comment:
Alpine *should* work, though I'm not 100% sure everything does. Is there really a big advantage over using the standard Python 3.9 images?

Reply:
Alpine is much smaller (Alpine is about 15MB, slim is about 40MB, and buster is 300MB).  For something thta's schlepped around in CI all the time, size does matter.  It also matters for people developing on bandwidth-limited connections (this is one of the major impediments for contributors to Firefox, for example, which requires several GB of data just to get started).

The Python env is the same, so the only differences would be in other files.  I don't think etesync calls out to external executables, limiting further to libraries and config files.  And that would include openssl and postgres (not used in a test image) and maybe miscellaneous config files like CA certs (again not used).

Reply:
We don't call external executable but we do use external libraries that may or may not fail. Anyhow, if it works let's keep it with alpine for now. We can always change it later.